src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -184,7 +185,7 @@ private async Task<string> GetLatestMatchingBundleVersion(HttpClient httpClient)
 
             if (string.IsNullOrEmpty(matchingBundleVersion))
             {
-                _logger.LogInformation(Resources.MatchingBundleNotFound, _options.Version);
```

Comment:
Can we remove the `MatchingBundleNotFound` string from resources? same with other strings that aren't needed anymore